packages/@expo/cli/src/start/server/metro/debugging/createDebugMiddleware.ts

```diff
@@ -16,7 +17,11 @@ export function createDebugMiddleware(metroBundler: MetroBundlerDevServer) {
     logger: createLogger(chalk.bold('Debug:')),
     unstable_customInspectorMessageHandler: createHandlersFactory(metroBundler),
     unstable_experiments: {
-      enableOpenDebuggerRedirect: true,
+      // Enable the Network tab in React Native DevTools
+      enableNetworkInspector: true,
```

Comment:
@Kudo we could also only enable this if `expo-dev-client` is installed. Mostly that if you don't have it, and it likely won't work, it's not visible

Reply:
it's tough to detect if running on expo-go mode, especially in dev-client mode, we have the interstitial page for people to launch in expo-go. for now on, let's keep it as-is to always enable network inspector.